astropy/uncertainty/core.py

```diff
@@ -220,6 +220,27 @@ def pdf_percentiles(self, percentile, **kwargs):
             return self.distribution._new_view(percs)
         else:
             return percs
+            
+    def pdf_asymunc(self, quantile_range=68):
```

Comment:
FWIW, I find `asymunc` a little cryptic. Is there a reason not to name this `pdf_async_uncertainty` or something that people can understand at a glance?

Reply:
@pllim  async?you meant asymm ,right?.i will change the name after @eteq  review it.

Reply:
Ah, yes, "asymmetric", not "async". Sorry for the confusion!

Reply:
well, it just shows that we shouldn't save a few characters out from names, better be implicit.